Update the introduction to UCASE() and UNHEX() (#16533)
pingcap/docs: functions-and-operators/string-functions.md

### [`UCASE()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_ucase)

The `UCASE()` function is used to convert a string to uppercase letters. This function is equivalent to the `UPPER()` function.

> **Note:**
>

> > When the string is `NULL`, the `UCASE()` function returns `NULL`.

Author: ```suggestion
> When the string is null, the `UCASE()` function returns `NULL`.
```

Example:

```sql

> SELECT upper('bigdata') AS result_upper, upper('null') AS result_null;

Reviewer: It seems that there is no `UCASE` in example.

> SELECT upper('bigdata') AS result_upper, upper('null') AS result_null;

Author: ```suggestion
SELECT UCASE('bigdata') AS result_upper, UCASE('null') AS result_null;
```

> SELECT UCASE('bigdata') AS result_upper, UCASE('null') AS result_null;

Author: To keep it consistent with the example for `UPPER()` https://github.com/pingcap/docs/pull/16534/files#r1502244394
```suggestion
SELECT UCASE('bigdata') AS result_upper, UCASE(null) AS result_null;
```

```

> Result:

Author: ```suggestion
Output:
```

```sql
+--------------+-------------+
| result_upper | result_null |
+--------------+-------------+
| BIGDATA      | NULL        |
+--------------+-------------+
```

### [`UNHEX()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_unhex)

The `UNHEX()` function performs the reverse operation of the `HEX()` function. It treats each pair of characters in the argument as a hexadecimal number and converts it to the character represented by that number, returning the result as a binary string.

> **Note:**
>

> > The argument must be a valid hexadecimal value that contains `0`-`9`, `A`-`F`, or `a`-`f`. If the argument is `NULL` or falls outside this range, the function returns `NULL`.

Author: ```suggestion
> The argument must be a valid hexadecimal value that contains `0`–`9`, `A`–`F`, or `a`–`f`. If the argument is `NULL` or falls outside this range, the function returns `NULL`.
```

> > The argument string must be a valid hexadecimal value that contains `0`~`9`, `A`~`F`, `a`~`f`. If the argument string is `NULL` or falls outside this range, it returns `NULL`.

Reviewer 2: ```suggestion
> The argument must be a valid hexadecimal value that contains `0`~`9`, `A`~`F`, or `a`~`f`. If the argument is `NULL` or falls outside this range, the function returns `NULL`.
```

> 

Reviewer 2: The GitHub preview looks weird. Let me set up a doc website preview. 

![Uploading image.png…]()

Reviewer 2: The website preview looks OK. 
https://pingcap-docsite-preview-git-pr-qiancai.vercel.app/tidb/dev/string-functions#unhex
![image](https://github.com/pingcap/docs/assets/79440533/7a92bf7b-0751-4c81-96c7-bc294f612377)

Author: @qiancai Thanks! I just tried another format using "-", which shows well on the GitHub page.

Example:

```sql
SELECT UNHEX('54694442');
```

> Result:

Author: ```suggestion
Output:
```
